layouts/partials/mermaid.html

```
@@ -1,12 +1,38 @@
-<script
-  type="application/javascript"
-  src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"
-></script>
 <script>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)